### ISSUE TITLE
Significant descrease in monster spawns

### ISSUE THREAD

**Issue by mhostilis** [Teneted/Tenet#972]
Hi! I own server with Mohist version 371. I've been dealing with very low spawn rates of monsters on my server. I thought this was a side affect of the Duplicate UUID issue which has now been fixed, but my issue with low spawn rates has persisted. I don't think there's anything in my server log that would indicate what the issue is, but here's a link just in case: https://pastebin.com/rNCKgcjQ

Here's a list of mods on the server: 
```
Appleskin 1.0.14
Aquaculture 2.1.12
Astikor Carts 1.1.0
AutoRegLib 1.16-47
Citadel 1.6.1
Cosmetic Armor Reworked 1.16.5-v1a
Create 0.3
JEI 7.6.1.68
Macaw's Bridges 1.0.5
Nature's Compass 1.8.6
ProjectE 1.0.0B
Quark 2.4-304
Rats 7.1.1
Serene Seasons 4.0.1.89
Storage Drawers 8.2.2
Towers of the Wild 2.0.1
Valhesia Structures 16.0.5
Waystones 7.3.1
```

And the plugins:
```
Discord Whitelister 1.4.5
Itemizer 5.01
PlayerKits 2.11.3
Shopkeepers 2.12.0
Worldedit 7.2.2
WorldGuard 7.0.4
```

Does anyone know what might be causing the issue? And is there any temporary workaround?

**Comment by Kowantify** [comment]
Spigot/Bukkit.yml, go into the config files and up the limits (don't get to crazy)

**Comment by Kowantify** [comment]
And it's not a "temporary workaround" that's just the default configuration involved with spigot and bukkit, so your actually bumping it up entirely, and u can also decrease the tick rate for monsters to spawn which will spawn them faster/more often

**Comment by mhostilis** [comment]
@KowalskiXCIX  Alrighty, I'll give that a try. Thank you!

**Comment by Kowantify** [comment]
Let me clarify something, I am not a dev for this software. I host multiple minecraft servers of all kinds, so i'm just sharing some knowledge that I have :)

**Comment by mhostilis** [comment]
> 
> 
> Let me clarify something, I am not a dev for this software. I host multiple minecraft servers of all kinds, so i'm just sharing some knowledge that I have :)

Well it seems to have worked for the most part! Although I did have to set the ticks per monster spawns to 50, which probably isn't ideal. It's also a little bit inconsistent in the fact that sometimes a lot of monsters will spawn, and sometimes very few, but it'll do the trick. Do you think I should leave this issue open since by default the spawns are low, or should I close it?

**Comment by Kowantify** [comment]
Spigot has those spawns set that low for massive servers, so it honestly isn't really an issue I would say, it's intended to be changed in the config, 50 is okay for the tick per spawn, I think 20 is the default for Minecraft but that's very low for servers and that can cause TPS issues, the lower it is the faster the "mob cap" will fill up as well, and mobs will quit spawning until some despawn/die. as long as you don't encounter TPS issues and your happy with it, you should be good to go

I would close the ticket as well since it really has nothing to do with mohist as a software, it's mostly just spigots default properties that are shipped with it.

**Comment by KR33PYK1NG** [comment]
I'm going to adjust default bukkit.yml values to more realistic in my next PR.
My guess is that mob tickrates were replaced with higher values to mitigate excessive mob spawns bug (which was later fixed in ffd5bd48287cb8fda36df5d57dab306935f20298)
